HW1/P2/P2b.py

```diff
@@ -1,3 +1,26 @@
 from P2 import *
+from pyspark import SparkContext
+import random
 
 # Your code here
+
+def partitionNode(node):
+    x,y = node
+    min_x = 800
+    max_x = 1000
+    min_y = 700
+    max_y = 1200
+    if x >= min_x and x < max_x and y >= min_y and y < max_y:
+        return (y-min_y)/5
+    else:
+        return random.randint(0, 99)
+
+sc = SparkContext("local", "P2a")
+rdd = sc.parallelize(xrange(0, 2000), 10)
```

Comment:
Could have used sc.range instead of creating the lists in python and then parallelizing them.